nh_observations/nh_clinical_extension.py

```diff
@@ -174,7 +174,9 @@ def trigger_notifications(self, cr, uid, values, context=None):
                 d_values = {
                     'patient_id': values.get('patient_id')
                 }
-                if n['model'] == 'frequency':
+                if n['model'] == 'frequency' \
```

Comment:
Cleaner to use `if n['model'] in ['frequency', 'select_frequency', 'weekly_frequency_agreed']` ?